site/src/pages/RoadmapPage/Year.tsx

```diff
@@ -90,7 +92,7 @@ const Year: FC<YearProps> = ({ yearIndex, data }) => {
               ) : (
                 <CaretRightFill className="caret-icon" />
               )}
-              <span id="year-number">Year {yearIndex + 1} </span>
+              <span id="year-number">{data.name} </span>
```

Comment:
The name attribute will not exist for user's with existing roadmaps. Given that, I think we should fall back to "Year {yearIndex + 1} in the event that the year does not have a name set. It displays nothing, otherwise.

Reply:
Changed to a conditional operator for data.name, that way if the user has a name for the year it wouldn't be "Year 1 Name".